### DISCUSSION TITLE
Grouping of projects

### DISCUSSION THREAD

**Discussion by kaiserkiwi** [spatie/flareapp.io-roadmap#52] [Ideas]
We often have multiple instances of a project that we track in Flare. Or both frontend and backend. Or just some projects for a specific customer. It would be great if these could be grouped together in the projects overview.

**Comment by AlexVanderbist** [comment]
Hi @kaiserkiwi, we recently added grouping for projects in different teams. We briefly discussed having arbitrary groups but ultimately decided against it because there just weren't that many teams with tons of projects. I'm guessing your team is one of the exceptions 😅 I'll raise this feature again with the rest of the team but can't make any promises. 

Thanks for your suggestion!

**Comment by AlexVanderbist** [comment]
Added groups anyways. Enjoy!
